tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import os
+
+root_dir = os.getcwd()
+module_name = "filterModule"
+
+@pytest.fixture(scope="module",params=["requirements.txt","module.json","main.py","Dockerfile.arm32v7","Dockerfile.amd64.debug","Dockerfile.amd64",".gitignore"])
```

Comment:
Suggest putting `params` definition outside of the decorator to make the code more readable.

Reply:
Updated code